src/modules/account/services/collections/collections.service.ts

```diff
@@ -61,6 +67,22 @@ class CollectionsService extends ApiService {
 			)
 			.json();
 	}
+
+	public async getExport(id?: string): Promise<Blob | null> {
+		if (!id) {
+			return null;
+		}
```

Comment:
Does this pass type-check? `null` =/= `Promise<null>`

Reply:
yes, any function with the `async` keyword will automatically output a Promise of whatever you return